polaris-core/src/main/java/org/apache/polaris/core/persistence/MetaStoreManagerFactory.java

```diff
@@ -34,6 +35,20 @@ public interface MetaStoreManagerFactory {
 
   BasePersistence getOrCreateSession(RealmContext realmContext);
 
+  /**
+   * Creates a new MetaStoreSession for the given realm. Use this when bootstrapping a realm or from
+   * tests. If you are handling an incoming request use @link #getMetaStoreSession(CallContext)
+   * instead.
+   */
+  MetaStoreSession createMetaStoreSession(RealmContext realmContext);
```

Comment:
I'd prefer to detach `MetaStoreSession` from `MetaStoreManagerFactory`. Can we produce sessions in a way that does not require all `MetaStoreManagerFactory` implementations to support a new method?

Reply:
`getMetaStoreSession()` is already called from `ServiceProducers` , but that call can be inlined.

Reply:
thx - LGTM. My main idea was to reduce change to core classes in this PR